src/client/Microsoft.Identity.Client/MsalServiceExceptionFactory.cs

```diff
@@ -138,6 +148,12 @@ private static bool IsInvalidGrant(string errorCode, string subErrorCode)
                              && IsInvalidGrantSubError(subErrorCode);
         }
 
+        private static bool IsPolicyProtectionRequired(string errorCode, string subErrorCode)
+        {
+            return string.Equals(errorCode, BrokerResponseConst.iOSBrokerProtectionPoliciesRequired, StringComparison.OrdinalIgnoreCase)
+                             && string.Equals(subErrorCode, MsalError.ProtectionPolicyRequired, StringComparison.OrdinalIgnoreCase);
```

Comment:
nit: RequiredAppProtectionPolicy?

Reply:
The actual err code is: "protection_policy_required". Keeping it based on that consistent with other constants.